marimo/_utils/health.py

```diff
@@ -19,7 +21,7 @@ def get_node_version() -> Optional[str]:
             stderr=subprocess.PIPE,
             text=True,
         )
-        stdout, stderr = process.communicate()
+        stdout, stderr = process.communicate(timeout=TIMEOUT)
```

Comment:
Should probably catch the error:

proc = subprocess.Popen(...)
try:
    outs, errs = proc.communicate(timeout=15)
except TimeoutExpired:
    proc.kill()
    return None